code-challenges.js

```diff
@@ -20,37 +20,74 @@ const fruit4 = "kiwi"
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 // Pseudo code:
+// Take an input number
+// Use a conditional statement to compare input numbers to temperature 212 degrees
+//  Return string statements to describe results
 
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+const boilChecker = (temp) => {
+ if (temp < 212) {
+    return `"${temp} is below boiling point"`}
+ else if (temp > 212) {
+    return `"${temp} is above boiling point"`}
+ else  (temp === 212) ;{
+    return `"${temp} is at boiling point"`}
+}
+console.log(boilChecker())
```

Comment:
Since you stated on line 31 that the function should expect a parameter, when you do the function call on line 39 you need an argument. You should have three different console.log's to test out all the conditionals.

`console.log(boilChecker(temp1))`
`console.log(boilChecker(temp2))`
`console.log(boilChecker(temp3))`